docs/cli-spec.md

```diff
@@ -1,29 +1,27 @@
 ## Polystat usages
 
 ### Notation
-`|` - exclusive or
-`(...)` - must be present
-`[...]` - may not be present
-`<thing>...` -- multiple instances of thing can be accepted.
+The description follows [this guide](https://en.wikipedia.org/wiki/Command-line_interface#:~:text=Command%20description%20syntax%5Bedit%5D).
+> Note: {a | b | c} means a set of _mutually-exclusive_ items.
 ```
-polystat (eo | java | python) --tmp <path> --in <path> [--include <rule>... | --exlude <rule>...] --sarif  --files [<path>]
-polystat --version --help
+polystat {eo | java | python} [--tmp <path>] [--in <path>] [{--include <rule...> | --exclude <rule...>}] [--sarif] [--files [path]]
```

Comment:
How about adding a `polystat config <file>` option? This will read the configuration from a file and won't interfere with the terminal-supplied commands. Or, config will be overriden by the terminal-supplied commands. What do you think?

Reply:
If some alternative location needs to be specified, `polystat --config <path>` can be used instead.